README.md

```diff
@@ -0,0 +1,91 @@
+# terraform-aws-nexus [![](https://github.com/rhythmictech/terraform-aws-nexus/workflows/pre-commit-check/badge.svg)](https://github.com/rhythmictech/terraform-aws-nexus/actions) <a href="https://twitter.com/intent/follow?screen_name=RhythmicTech"><img src="https://img.shields.io/twitter/follow/RhythmicTech?style=social&logo=RhythmicTech" alt="follow on Twitter"></a>
+
+Create a Sonatype IQ Server instance. This does some neat things:
+
+* `sonatype-work` directory is managed by EFS with optional backups using AWS Backup
+* everything runs in an ASG (though HA isn't supported.. yet..) so if something happens to the instance, it'll come back up automatically.
+* updates are done by upgrading your AMI and replacing the launch config
+* automatically manages licensing
+
+## Requirements
+
+This expects an instance that has IQ Server pre-installed using the Rhythmic [ansible-role-iqserver](https://github.com/rhythmictech/ansible-role-iqserver) ansible module. The easiest way to get one is to use Packer.
+
+## License File
+To use auto licensing, you need to save your license file in AWS Secrets Manager. Something like this would work:
+
+```
+aws --region us-east-1 secretsmanager create-secret --secret-id iqserver-license --secret-binary=file:///tmp/license.lic
+```
+
+_Tip: when you renew your license, update the secret and kill the instance. It will automatically be updated._
+
+## Example
+Here's what using the module will look like
+```
+module "example" {
+  source = "git::https://github.com/rhythmictech/terraform-aws-iqserver.git"
+
+  name                           = "nexus"
+  ami_id                         = "ami-12345678912"
+  asg_subnets                    = ["subnet-123456789012", "subnet-123456789013"]
+  efs_subnets                    = ["subnet-123456789012", "subnet-123456789013"]
+  elb_certificate                = "arn:aws:acm:us-east-1:12345678912:certificate/090c1a21-f053-4aac-8b92-2c963c3c0660"
+  elb_subnets                    = ["subnet-123456789012", "subnet-123456789013"]
+  vpc_id                         = "vpc-123456789012"
+}
+```
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+## Requirements
+
+| Name | Version |
+|------|---------|
+| terraform | >= 0.12.19 |
```

Comment:
This doesn't match the `.terraform-file` I'll put a suggestion to lower that file